README.md

```diff
@@ -30,7 +30,7 @@ After you're finished please remove all the comments and instructions!
   <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
- [optional] please kindly remove the Microverse logo from the README. You can keep the space empty until you have a logo to use.

Reply:
Fixed.